frontend/public/ui/ui-reducers.js

```diff
@@ -5,6 +5,7 @@ import { types } from './ui-actions';
 import { ALL_NAMESPACES_KEY, LAST_NAMESPACE_NAME_LOCAL_STORAGE_KEY, NAMESPACE_LOCAL_STORAGE_KEY } from '../const';
 import { AlertStates, isSilenced, SilenceStates } from '../monitoring';
 import { legalNamePattern, getNamespace } from '../components/utils/link';
+import { GROUP_BY_APPLICATION } from '../components/overview';
```

Comment:
This import will pull overview into the main bundle

Reply:
Moved the shared constants into a separate `constants.ts` file